storage/remote/queue_manager.go

```diff
@@ -204,17 +241,19 @@ func (m *queueManagerMetrics) register() {
 			m.maxNumShards,
 			m.minNumShards,
 			m.desiredNumShards,
-			m.bytesSent,
-			m.maxSamplesPerSend,
+			m.commonBytesVector,
```

Comment:
I don't think this is used anymore, but you now need to register `m.samplesBytesTotal` and `m.metadataBytesTotal`?

Reply:
Nevermind, github told me to refresh and you already fixed it :)